pootle/templates/editor/units/edit.html

```diff
@@ -152,19 +152,7 @@
                 </div>
                 {% endif %}
               </div>
-              <div id="js-unit-{{ unit.id }}"
-                   class="translate-original{% if unit.hasplural %} translate-plural{% endif %}">
-                {% for i, source, title in unit|pluralize_source %}
-                <div
-                  class="translation-text js-translation-text"
-                  lang="{{ source_language.code }}"
-                  dir="{{ source_language.direction }}"
-                  {% if title %} title="{{ title }}"{% endif %}
-                  data-string="{{ source|escapejs }}"
-                ></div>
-                {% endfor %}
-                <div class="placeholder"></div>
-              </div>
+              <div class="js-mount-editor-original"></div>
```

Comment:
Alternative source language texts are a set of source texts and they have exactly the same layout as the source strings in the original language. Therefore alternative source language texts need to use this component too.